registry/remote/example_test.go

```diff
@@ -49,6 +50,18 @@ var (
 	exampleManifestDigest = digest.FromBytes([]byte(exampleManifest)).String()
 )
 
+var (
+	exampleReferenceManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example reference manifest",
+		Subject:      artifactspec.Descriptor{Digest: ocidigest.Digest(exampleManifestDigest)}})
+	exampleReferenceManifestDescriptor = ocispec.Descriptor{
```

Comment:
Should use `artifactspec.MediaTypeArtifactManifest`

Reply:
changed.